spec/unit/auto_registrar_spec.rb

```diff
@@ -22,6 +23,20 @@
     expect(container["foo"]).to be_an_instance_of(Test::Foo)
     expect(container["bar"]).to be_an_instance_of(Test::Bar)
     expect(container["bar.baz"]).to be_an_instance_of(Test::Bar::Baz)
+
+    expect { container["bar.abc"] }.to raise_error(
+      Dry::System::ComponentNotLoadableError
+    ) do |error|
+      expect(error.message).to eq(
```

Comment:
Should I get rid of this message expectation? We already test the message in the new `spec/unit/dry/system/errors_spec.rb` file